mobile/src/main/AndroidManifest.xml

```diff
@@ -8,6 +8,7 @@
     <uses-permission
             android:name="android.permission.PACKAGE_USAGE_STATS"
             tools:ignore="ProtectedPermissions"/>
+    <meta-data android:name="com.samsung.android.vr.application.mode" android:value="dual"/>
```

Comment:
Not sure if this will be enough to have it show up in the Oculus TV menu on Quest, but we'll see.

Reply:
Seems like I also might want to add an entry for opening it on a TV: https://developer.android.com/training/tv/start/start#tv-activity